platform/commonUI/general/src/directives/MCTPopup.js

```diff
@@ -51,6 +51,8 @@ define(
                     position = [ rect.left, rect.top ],
                     popup = popupService.display(div, position);
 
+                div.css('z-index', 75);
```

Comment:
Could we include this style in a class? The class could then be applied from code if need be. Worry that setting styles directly from code adversely affects maintainability. Especially with z-index which is something that is clearly being carefully managed.